package.json

```diff
@@ -152,13 +152,15 @@
     "wherearewe": "^2.0.1"
   },
   "devDependencies": {
+    "@libp2p/interfaces": "^3.0.3",
```

Comment:
This dep is added but doesn't appear to be used anywhere?

```suggestion
```

Reply:
When removing this, ipfs-core-types throws an error

Reply:
@SgtPooki : a couple of thoughts:
1. Do you want to paste in the error observed to aid in understanding or debugging?
2. assuming getting this merged is blocking other work you're trying to do for the js-kubo-rpc-client, I think it's safe to merge and then we fast-followup if there's something deeper to address that @achingbrain calls out.

Reply:
pushing up a change removing this to see if CI still passes.